sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -81,6 +81,13 @@ private TestProxy(string proxyPath, bool debugMode = false)
                 }
             };
 
+            bool.TryParse(Environment.GetEnvironmentVariable("TF_BUILD"), out bool inCI);
```

Comment:
The changes from this PR can be reverted when https://github.com/Azure/azure-sdk-tools/issues/6562 is fixed and deployed.